### PR TITLE
ARTEMIS-977 - record id of address not set on journal reload

### PR DESCRIPTION
**Pull request by andytaylor**
https://issues.apache.org/jira/browse/ARTEMIS-977

### CODE REVIEW

File: tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/ActiveMQServerControlTest.java [hunk 1]
```diff
@@ -412,6 +412,27 @@ public void testGetAddressNames() throws Exception {
    }
 
    @Test
+   public void testGetAddressDeletedFromJournal() throws Exception {
```
Comment (jdanekrh) [inline tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/ActiveMQServerControlTest.java]: This test is passing for me even without the change in PostOfficeJournalLoader.java.

I wanted to figure out why I was not able to reproduce this with JUnit test the day before yesterday and I noticed what I said in the first paragraph. Do you have any ideas what might be wrong?

**Comment by andytaylor** [comment]
dont merge this just yet

**Comment by andytaylor** [comment]
ok, good to go

File: tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/ActiveMQServerControlTest.java (hunk 1 above)
Comment (jdanekrh) [inline tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/ActiveMQServerControlTest.java]: Now it is working for me as it should.

**Comment by clebertsuconic** [comment]
I'm running tests to make sure before merging

**Comment by clebertsuconic** [comment]
there seems to be a checkstyle issue?

**Comment by andytaylor** [comment]
I didnt see an issue when i built locally

**Comment by clebertsuconic** [comment]
There's actually a checkstyle issue. It's a minor thing. a space on a line.


if you can't fix it this monday I will amend myself and push it.

**Pull request closed without merging**